test/docbook-reader.native

```diff
@@ -10,6 +10,8 @@ Pandoc (Meta {unMeta = fromList [("author",MetaList [MetaInlines [Str "John",Sof
 ,Header 2 ("level-2-with-emphasis",[],[]) [Str "Level",Space,Str "2",Space,Str "with",Space,Emph [Str "emphasis"]]
 ,Header 3 ("level-3",[],[]) [Str "Level",Space,Str "3"]
 ,Para [Str "with",Space,Str "no",Space,Str "blank",Space,Str "line"]
+,Header 4 ("",["unnumbered"],[]) []
```

Comment:
is this empty Header intended? HTML output e.g. would be

    <h4 class="unnumbered" class="unnumbered"></h4>